README.md

```diff
@@ -285,6 +285,7 @@ The following table describes each configurable parameter in the VerticaDB CRD a
 | ignoreClusterLease | Ignore the cluster lease when doing a revive or start_db.  Use this with caution, as ignoring the cluster lease when another system is using the same communal storage will cause corruption. | false
 | kSafety | Sets the fault tolerance for the cluster. Allowable values are 0 or 1. 0 is only suitable for test environments because we have no fault tolerance and the cluster can only have between 1 and 3 pods. If set to 1, we have fault tolerance if nodes die and the cluster has a minimum of 3 pods.<br>This value cannot change after the initial creation of the VerticaDB.| 1 |
 | reviveOrder | This specifies the order of nodes when doing a revive.  Each entry contains an index to a subcluster, which is an index in `subclusters[i]`, and a pod count of the number of pods include from the subcluster.<br><br>For example, suppose the database you want to revive has the following setup:<br>- v_db_node0001: subcluster A<br>- v_db_node0002: subcluster A<br>- v_db_node0003: subcluster B<br>- v_db_node0004: subcluster A<br>- v_db_node0005: subcluster B<br>- v_db_node0006: subcluster B<br><br>And the `subclusters[]` list is defined as {'A', 'B'}.  The revive order would be:<br>- {subclusterIndex:0, podCount:2}  # 2 pods from subcluster A<br>- {subclusterIndex:1, podCount:1}  # 1 pod from subcluster B<br>- {subclusterIndex:0, podCount:1}  # 1 pod from subcluster A<br>- {subclusterIndex:1, podCount:2}  # 2 pods from subcluster B<br><br>If InitPolicy is not Revive, this field can be ignored.|Not set
+| restartTimeout | This specifies the timeout to use calling admintools to restart pods.  If not specified, it defaults 0, which means we will use the admintools default of 20 minutes. | 0 |
```

Comment:
Minor: this is readme so it would be nice to mention that it's in seconds, but this is very minor as AT specifies timeout in seconds so users could guess that it's also in seconds.

Reply:
Good point, I will include seconds.